sale_commission_pricelist/models/__init__.py

```diff
@@ -1,3 +1,3 @@
 
-from . import product_pricelist
+from . import product_pricelist_item
```

Comment:
Well, the name was put as the main model in product.pricelist, not product.pricelist.item, so it contains possible changes to main model and also changes on children models.

Reply:
I'm not sure what you mean, does it mean that i should keep the old name?

Reply:
Yes, not mandatory, but it was done that way by purpose, so no need of change it.